packages/client-core/src/user/components/UserMenu/menus/ProfileMenu.tsx

```diff
@@ -500,10 +501,36 @@ const ProfileMenu = ({ hideLogin, onClose, isPopover }: Props): JSX.Element => {
                   control={
                     <Checkbox
                       disabled={hasAcceptedTermsAndAge}
+                      value={checked13OrOver.value}
+                      onChange={(e) => checked13OrOver.set(e.target.checked)}
+                      color="primary"
+                      name="is13OrOver"
+                    />
+                  }
+                  label={
+                    <div
+                      style={{
+                        fontStyle: 'italic'
+                      }}
+                      className={styles.termsLink}
+                    >
+                      {t('user:usermenu.profile.confirmAge13')}
+                    </div>
+                  }
+                />
+              </Grid>
+            )}
+
+            {!isGuest && !originallyAcceptedTOS.value && (
```

Comment:
It seems like this is a fallback for users who existed before this PR, right? Because I normally don't see this if I start as a new guest user, accept TOS and verify age, and then log in. But I am able to make it appear if I set the acceptedTOS column to false for a non-guest user. Fine if it is, just thinking out loud mostly.